website/docs/r/nsxt_app_port_profile.html.markdown

```diff
@@ -69,10 +72,14 @@ The following arguments are supported:
 
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
   when connected as sysadmin working across different organisations.
-* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level.
+* `vdc` - (Deprecated; Optional) The name of VDC to use, optional if defined at provider level.
+  Deprecated and replaced by `context_id`
+* `context_id` - (Optional) ID of NSX-T Manager, VDC or VDC Group. Replaces deprecated fields `vdc`
+  and `nsxt_manager_id`. It accepts VDC, VDC Group or NSX-T Manager ID. 
 * `name` - (Required) A unique name for Security Group
 * `scope` - (Required) Application Port Profile scope - `PROVIDER`, `TENANT`
-* `nsxt_manager_id` - (Optional) Required only when `scope` is `PROVIDER`
+* `nsxt_manager_id` - (Deprecated; Optional) Required only when `scope` is `PROVIDER`. Deprecated
+  and replaced by `context_id`
 * `app_port` - (Required) At least one block of [Application Port definition](#app-port)
 
```

Comment:
documentation doesn't have details and I confirmed that it isn't possible to import using vdcGroup

Reply:
Indeed. Because the application port profiles are not bound to VDC Groups. (Whatever you create - you can see it in Edge Gateway and in VDC Group). But I will adjust this to support importing by VDC Group name just for having it more convenient.

Reply:
Indeed. Because the application port profiles are not bound to VDC Groups. (Whatever you create - you can see it in Edge Gateway and in VDC Group). But I will adjust this to support importing by VDC Group name just for having it more convenient.

Reply:
I have improved the code to handle VDC group name explicitly and tests.